docs/project/contribute.md

```diff
@@ -64,13 +64,39 @@ Search is by algolia, the configuration is at <https://github.com/algolia/docsea
 
 ## Running the site locally:
 
-All Contributors uses Docusaurus to maintain our documentation website. For more info visit the [Docusaurus Website](https://docusaurus.io)
+All Contributors currently uses [Docusaurus 1.x](https://docusaurus.io) for our documentation website.
 
-To get started locally:
+To build the docs locally:
 
-- `yarn install`
-- `yarn start`
-- go to `http://localhost:3000` (Refresh the page for your changes to be reflected)
+* https://tutorial.docusaurus.io/docs/intro
+
+1. Make sure `npm` is installed on your machine. Use the [nodejs documentation](https://docs.npmjs.com/downloading-and-installing-node-js-and-npm) if you aren't sure how to install `npm`. 
+
+To check that npm is installed run the following in your favorite shell:
+
+`npm -v`
+
+Next, make sure you have atleast node version 0.18 or higher. To check the version,
+use:
+
+`node -v`
+
+If you don't have atleast version `.18` or higher, please install node or upgrade your current version.
```

Comment:
If this is for the Node version, v24 should be the oldest to accept, as it's the current LTS version. v18 has been out of support for several years.

Reply:
Definitely. The entire docs website is out of date with a lot of deprecated dependencies. I will test this locally and then update!! Great catch!